README.md

```diff
@@ -110,7 +151,26 @@ yarn start
 
 > Find the `packages/snfoundry/.env` file and fill the env variables related to Sepolia testnet with your own contract address and private key.
 
-⛽️ You will need to get some `ETH` or `STRK` Sepolia tokens to deploy your contract to Sepolia testnet.
+>  How to Install the example Wallet on Browser
```

Comment:
Here we need to simplify the process, something like this [Setting up an Account](https://docs.starknet.io/quick-start/set-up-an-account/#export_the_private_key_from_your_wallet) and/or with the links to download the wallets

![Captura desde 2024-08-29 10-31-20](https://github.com/user-attachments/assets/380ded7f-0d12-415c-a606-e3ecfef5116c)

We also need to add the two main wallets at least, the two tokens `Strk` or `Eth` and simplified faucet options [similar to Scaffold Stark](https://github.com/Quantum3-Labs/scaffold-stark-2?tab=readme-ov-file#quickstart-with-sepolia-testnet)